exec/java-exec/src/main/codegen/templates/StringOutputRecordWriter.java

```diff
@@ -146,7 +146,7 @@ public void writeField() throws IOException {
     // TODO: error check
     addField(fieldId, reader.readObject().toString());
 
-  <#elseif minor.class == "VarChar" || minor.class == "Var16Char" || minor.class == "VarBinary">
+  <#elseif minor.class == "VarChar" || minor.class == "Var16Char" || minor.class == "VarBinary" || minor.class == "VarDecimal">
```

Comment:
May these types be moved into previous `elseif`?

Reply:
Yes, that works